docs/HTTPSCertificates.md

```diff
@@ -0,0 +1,24 @@
+# Managing nginx HTTPS Certificates
+
+nginx needs to provide secure HTTP communication between clients and servers. This means that the server needs to be able to verify itself to the client so that the client is confident it's sending messages to the correct place, and that the client can communicate in a secure fashion with this server. nginx achieves this through HTTPS (HTTP over SSL). This involves the server sending a signed certifcate (or public key) to the client, who is able to verify the signature. The client may also request the certificate authority (CA) certificate signing request (CSR) which was used to sign the certificate. Once the client has verified the signed certificate, it can use it to securely send a means of encryption for subsequent messages to the server. The server can read this using the private key, and the server and client can now exchange information securely. There are three options for generating the secure certificate.
```

Comment:
You seem to have some confusion between CSR, CA and Public/Private certificates.  This leads to issues throughout.  Grab me for a chat if you need to.